awx/main/tasks/system.py

```diff
@@ -86,6 +88,8 @@ def dispatch_startup():
     if settings.IS_K8S:
         write_receptor_config()
 
+    convert_jsonfields()
```

Comment:
I've debugged too many things that error in this startup method, I would like to have this in a blanket try-except. I don't ever want a failure here to prevent dispatcher startup.

Reply:
fair enough